WordPress/Classes/System/WordPressAppDelegate.m

```diff
@@ -571,6 +571,7 @@ - (void)customizeAppearance
 
     [[UINavigationBar appearance] setBackgroundImage:[UIImage imageWithColor:[WPStyleGuide wordPressBlue]] forBarMetrics:UIBarMetricsDefault];
     [[UINavigationBar appearance] setShadowImage:[UIImage imageWithColor:[UIColor UIColorFromHex:0x007eb1]]];
+    [[UINavigationBar appearance] setBarStyle:UIBarStyleBlack];
```

Comment:
When the status bar bellows to a navigation bar, the status bar style is adjusted to match the navigation bar style. So, if the navigation bar style is ```UIBarStyleDefault``` then the status bar style is ```UIStatusBarStyleDefault```, if the navigation bar style is ```UIBarStyleBlack``` (our case) the status bar style will be ```UIStatusBarStyleLightContent```.